sml_builder/templates/cookies.html

```diff
@@ -36,47 +36,123 @@ <h2>Cookie settings</h2>
+                                        },
+                                        {
+                                            "value": "10 minutes"
+                                        }
+                                    ]
+                                }
+                            ]
+                        })
+                    }}
+                {% endcall %}
+                <p>We do not allow Google to use or share the data about how you use this site.</p>
+                <div class="ons-u-mb-s">
+                    {{ onsRadios({
+                        "id": "cookies-measure-website-use",
+                        "name": "cookies-usage",
+                        "legend": "Do you want to allow usage tracking?",
+                        "legendClasses": "ons-u-vh",
+                        "radios": [
+                            {
+                                "id": "on-measure-website-use",
+                                "label": {
+                                    "text": "On"
+                                },
+                                "value": "on"
                             },
-                            "value": "off"
-                        }
-                    ]
-                }) }}
+                            {
+                                "id": "off-measure-website-use",
+                                "label": {
+                                    "text": "Off"
+                                },
+                                "value": "off"
+                            }
+                        ]
+                    }) }}
+                </div>
+                <p>This selection will expire after two years.</p>
+
                 <h3>Strictly necessary cookies</h3>
                 <p>These essential cookies do things like:</p>
                 <ul>
-                    <li>remember the notifications you've seen, so we do not show them to you again</li>
+                    <li>remember the notifications you've seen so we do not show them to you again</li>
                     <li>remember your progress through a form (for example, an ons study)</li>
                 </ul>
                 <p>They always need to be on.</p>
-                {%- call onsDetails({
+                {% call onsDetails({
                     "id": "cookies-essential-details",
                     "classes": "ons-u-mb-l",
                     "title": "Learn more about these essential cookies",
-                    "titleTag": "h4",
+                    "titleTag": 'h4',
                     "button": {
                         "close": "Hide this"
                     }
-                }) -%}
-                    <h4>Your progress when filling in a survey or questionnaire</h4>
+                }) %}
+                    <h5 class="ons-u-fs-r--b">Your progress when filling in a survey or questionnaire</h5>
```

Comment:
We don't have any surveys or questionnaires so we should remove this.

Reply:
Removed :)